packages/app/src/app/pages/NewDashboard/Components/Selection/ContextMenu.tsx

```diff
@@ -342,6 +370,35 @@ const SandboxMenu: React.FC<SandboxMenuProps> = ({ item, setRenaming }) => {
   );
 };
 
+const RepoMenu = ({ repo }) => {
+  const { effects } = useOvermind();
+  const { visible, setVisibility, position } = React.useContext(Context);
+  const link = `https://github.com/${repo.owner}/${repo.name}/tree/${repo.branch}`;
+  const csbLink = `https://codesandbox.io/s/github/${repo.owner}/${repo.name}/tree/${repo.branch}`;
```

Comment:
For generating links it would be better to use the helper function. This wouldn't work for example for sandboxes that are linked to a commit. in `url-generators.ts` there is a `sandboxLink` (or similar name) function that handles the `repo` info.

Reply:
Done :)